onedal/decomposition/incremental_pca.py

```diff
@@ -143,20 +144,25 @@ def partial_fit(self, X, queue):
         else:
             self.n_components_ = self.n_components
 
-        module = self._get_backend("decomposition", "dim_reduction")
-
-        if not hasattr(self, "_policy"):
-            self._policy = self._get_policy(queue, X)
+        if not hasattr(self, "_queue"):
+            self._queue = queue
```

Comment:
Should be removed at `finalize_fit` I think

Reply:
i think the better way is to update it in every call of partial_fit, I'll do like that for all algos